### PR TITLE
Generate release changelog

### PR DESCRIPTION
**Pull request by gallettilance**
**Description of the change:**

**Motivation for the change:**

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

**Comment by jpeeler** [comment]
This looks messed up. The changelog is supposed to be rolling.

**Comment by gallettilance** [comment]
Ah nice catch!  I think I messed up the `from` and `to`. I'll update the docs too

File: CHANGELOG.md [hunk 1]
```diff
@@ -703,4 +770,6 @@ Other changes:
 
 
 
-\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
\ No newline at end of file
+\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```
Comment (jpeeler) [inline CHANGELOG.md]: I think we just want one entry? Can do as a follow up if you feel like it.

**Pull request merged**